### PR TITLE
Add logging to LoadBalancedFargateService

### PR DESCRIPTION
**Pull request by SoManyHs**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts [hunk 1]
```diff
@@ -101,6 +102,12 @@ export interface LoadBalancedFargateServiceProps {
    * Setting this option will set the load balancer port to 443.
    */
   certificate?: CertificateRef;
+  /**
+   * Allows opting out of creation of aws log drivers
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: I would just write "whether to add AWS log drivers"

File: packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts [hunk 2]
```diff
@@ -101,6 +102,12 @@ export interface LoadBalancedFargateServiceProps {
...
+   *
+   * @default true
+   */
+  createLogDriver?: boolean;
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: Should we mention it's the AWS one?

File: packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts [hunk 3]
```diff
@@ -121,8 +128,11 @@ export class LoadBalancedFargateService extends cdk.Construct {
       cpu: props.cpu
     });
 
+    const optIn = props.createLogDriver !== undefined ? props.createLogDriver : true;
+
     const container = taskDefinition.addContainer('web', {
       image: props.image,
+      logging: optIn === true ? this.addLogDriver(this.id) : undefined
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: You don't have to compare a boolean to `true` to test it. You can just go:

```
logging: optIn ? this.addLogDriver(this.id) : undefined
```

File: packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts [hunk 4]
```diff
@@ -175,4 +185,8 @@ export class LoadBalancedFargateService extends cdk.Construct {
       });
     }
   }
+
+  private addLogDriver(prefix: string): AwsLogDriver {
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: `addLogDriver` doesn't actually add anything to anything. It just returns a new logdriver.

**Comment by rix0rrr** [comment]
Approved modulo naming.

File: packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts (hunk 2 above)
Comment (clareliguori) [reply packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: "Log driver" feels too low level. I can know how to run a docker image locally without having any idea what log drivers are. Maybe just "logging"?

Comment (SoManyHs) [reply packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: I had `logging` originally but didn't want that to be confused with the `logging` field on the `ContainerDefinition` construct. I think `enableLogs` is better but wanted to be explicit that we're actually creating a LogGroup resource.

File: packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts (hunk 1 above)
Comment (SoManyHs) [reply packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: done

File: packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts (hunk 3 above)
Comment (SoManyHs) [reply packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts]: oops. done.

**Pull request merged**